src/Core/Components/Label/FluentInputLabel.razor

```diff
@@ -1,15 +1,17 @@
-@namespace Microsoft.FluentUI.AspNetCore.Components
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
 @inject LibraryConfiguration LibraryConfiguration
 
 
 @if (!string.IsNullOrEmpty(Label) || ChildContent is not null)
 {
-    <label for="@ForId" class="fluent-input-label" @attributes="@AdditionalAttributes" aria-label="@AriaLabel">
+    var orientation = Orientation == Orientation.Horizontal ? Orientation.ToAttributeValue() : null; 
+
+    <label for="@ForId" class="fluent-input-label" orientation=@orientation @attributes="@AdditionalAttributes" aria-label="@AriaLabel">
```

Comment:
A detail, but I think it could be better to always use double-quote for all attributes and the variable must start with an upper-case.

```xml
`orientation="@Orientation"`
```

Reply:
quotes added. In this case it is a local var (see line above). I added an '_' to make that distinction clearer